docs/src/examples/components/Popup/Types/PopupExample.shorthand.tsx

```diff
@@ -1,6 +1,19 @@
 import * as React from 'react'
-import { Button, Popup } from '@stardust-ui/react'
+import { Button, Popup, Dialog, popupFocusTrapBehavior } from '@stardust-ui/react'
 
-const PopupExample = () => <Popup trigger={<Button icon="expand" />} content="Hello from popup!" />
+const PopupExample = () => (
+  <Popup
+    trigger={<Button icon="expand" />}
+    accessibility={popupFocusTrapBehavior}
+    content={
+      <Dialog
+        cancelButton="Cancel"
+        confirmButton="Confirm"
+        header="Action confirmation"
+        trigger={<Button content="Open a dialog" />}
+      />
+    }
+  />
+)
```

Comment:
Should we revert these changes?

Reply:
oops, yeah. Did some testing, great you noticed